tests/foreman/ui/test_host.py

```diff
@@ -447,6 +448,67 @@ def test_positive_export(session, target_sat, function_org, function_location):
         assert set(actual_fields) == expected_fields
 
 
+@pytest.mark.skipif(
+    (settings.ui.webdriver != 'chrome'), reason='Currently only chrome is supported'
+)
+@pytest.mark.tier3
+def test_positive_export_selected_columns(target_sat, current_sat_location):
```

Comment:
Can you add  module org and module loc?

Reply:
I am not sure it's needed. This test needs just the Sat. host, so there is at least one host displayed in the table so it can be exported. No additional hosts need to be created for this test.
Or did you mean something else with the module org. and loc., @shweta83 ?

Reply:
@shweta83 can we resolve this so the PR can be merged soon? thanks!

Reply:
No, we can resolve this.